ntbs-service/Pages/Alerts/TransferRequest.cshtml

```diff
@@ -15,7 +15,7 @@
     <form method="post" autocomplete="off">
         <h2> You are about to transfer this notification </h2>
         <div>
-            <filtered-dropdown filter-handler-path="FilteredCaseManagersListByTbServiceCode" :filtering-refs="['caseManagers']" inline-template>
+            <filtered-dropdown filter-handler-path="FilteredCaseManagersListByTbServiceCode" :filtering-refs="['caseManagers']" :wait-for-child-mount="false" inline-template>

```

Comment:
It seems weird that by not waiting for something we're now waiting longer for the case managers to load! Could you comment a little more on what's going on here?